test/Common/TestDurableClient.cs

```diff
@@ -182,7 +182,7 @@ public async Task<DurableOrchestrationStatus> WaitForStartupAsync(ITestOutputHel
         {
             if (timeout == null)
             {
-                timeout = Debugger.IsAttached ? TimeSpan.FromMinutes(5) : TimeSpan.FromSeconds(30);
+                timeout = Debugger.IsAttached ? TimeSpan.FromMinutes(5) : TimeSpan.FromMinutes(1);
```

Comment:
GitHub actions  + Azurite seems to run tests more slowly, so I added more buffer time